python/manylinux1/scripts/build_clang.sh

```diff
@@ -18,8 +18,8 @@
 
 source /multibuild/manylinux_utils.sh
 
-export LLVM_VERSION="7.0.1"
-curl -sL https://releases.llvm.org/${LLVM_VERSION}/cfe-${LLVM_VERSION}.src.tar.xz -o cfe-${LLVM_VERSION}.src.tar.xz
+export LLVM_VERSION="8.0.1"
+curl -sL https://github.com/llvm/llvm-project/releases/download/llvmorg-${LLVM_VERSION}/cfe-${LLVM_VERSION}.src.tar.xz -o cfe-${LLVM_VERSION}.src.tar.xz
```

Comment:
@kou some of the changes in this were not cherry-picked, is there a reason?

Reply:
Oh, sorry.
It was my mistake.

We miss only https://github.com/apache/arrow/pull/6394/commits/c0a2b65a897b3801b1474269c530c09d3120d87c ?

Reply:
#6651

Reply:
looks like it but we cannot directly cherry-pick c0a2b65 now - only changes which were not picked should be added now

Reply:
i will reopen this and rebase this branch on master